services/aur/aur.service.js

```diff
@@ -118,6 +119,36 @@ class AurVotes extends BaseAurService {
   }
 }
 
+class AurPopularity extends BaseAurService {
+  static category = 'rating'
+
+  static route = { base: 'aur/popularity', pattern: ':packageName' }
+
+  static openApi = {
+    '/aur/popularity/{packageName}': {
+      get: {
+        summary: 'AUR Popularity',
+        description: 'Arch linux User Repository Popularity',
+        parameters: pathParams({ name: 'packageName', example: 'dropbox' }),
+      },
+    },
+  }
+
+  static defaultBadgeData = { label: 'popularity' }
+
+  static render({ popularity }) {
+    return {
+      message: metric(popularity),
+      color: floorCountColor(popularity, 0.5, 2.5, 5),
+    }
+  }
+
+  async handle({ packageName }) {
+    const json = await this.fetch({ packageName })
+    return this.constructor.render({ votes: json.results[0].Popularity })
```

Comment:
```suggestion
    return this.constructor.render({ popularity: json.results[0].Popularity })
```

`render()` does not take a parameter called `votes`
That is why every badge says `popularity | undefined`

Reply:
Right! I initially copied from the `votes` and changed it all to `popularity`. I missed this one.